plugins/funind/glob_termops.mli

```diff
@@ -122,3 +122,10 @@ val zeta_normalize : Glob_term.glob_constr -> Glob_term.glob_constr
 
 
 val expand_as : glob_constr -> glob_constr
+
+
+(* [resolve_and_replace_implicits ?expected_type env sigma rt] solves implicits of [rt] w.r.t. [env] and [sigma] and then replace them by their solution 
+ *)
+val resolve_and_replace_implicits :
```

Comment:
I second @forestjulien on this: one of the good features of functional induction is that when the user uses wildcard in his pattern matching the induction scheme also does.